__test__/task.test.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * @jest-environment jsdom
+ */
+
+import { addTask, deleteTask, taskList } from '../src/module/task.js';
+
+// Tests for addTask function
+describe('addTask function', () => {
+  test('should add a new task to the task list', () => {
+    addTask('Task 2');
+    expect(taskList.children.length).toBe(2);
+    expect(taskList.children[1].querySelector('.task-desc').textContent).toBe('Task 2');
+  });
+  test('should not add a task if the description is empty', () => {
+    addTask('');
+    expect(taskList.children.length).toBe(2);
+  });
+});
+// Tests for deleteTask function
+describe('deleteTask function', () => {
+  test('should delete a task from the task list', () => {
+    const deleteButton = taskList.children[0].querySelector('.delete-task-button');
+    const taskItem = deleteButton.parentElement;
+    deleteTask(taskItem);
+    expect(taskList.children.length).toBe(1);
+  });
+  test('should not delete a task if the index is out of range', () => {
+    deleteTask(taskList.children.length);
+    expect(taskList.children.length).toBe(1);
+  });
+});
```

Comment:
While you did a good job working on the project requirements , when I run your tests I notice its throwing this error
![Screenshot 2023-03-09 at 9 18 55 PM](https://user-images.githubusercontent.com/79506105/224132080-03207631-b07c-49a3-a9ff-cd6b23019118.png)

It would be nice if you corrected this error for your tests to pass .

Reply:
@KayLemba  Is there a way I can preinstall in for whoever clones the repo?
![Screenshot 2023-03-09 153608](https://user-images.githubusercontent.com/118696200/224135380-0c5f2ccd-429c-47bb-ab9c-032e7cf57cc2.png)